minio/java/src/main/java/cloud/benchflow/minio/BenchFlowMinioClient.java

```diff
@@ -0,0 +1,371 @@
+package cloud.benchflow.minio;
+
+import cloud.benchflow.commons.hashing.Hashing;
+import io.minio.ErrorCode;
+import io.minio.MinioClient;
+import io.minio.ObjectStat;
+import io.minio.Result;
+import io.minio.errors.*;
+import io.minio.messages.Item;
+import org.apache.commons.io.IOUtils;
+import org.xmlpull.v1.XmlPullParserException;
+
+import java.io.ByteArrayInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.UnsupportedEncodingException;
+import java.nio.charset.StandardCharsets;
+import java.security.InvalidKeyException;
+import java.security.NoSuchAlgorithmException;
+import java.util.LinkedList;
+import java.util.List;
+
+/**
+ * @author Simone D'Avico (simonedavico@gmail.com)
+ *
+ * Created on 26/02/16.
+ */
+@SuppressWarnings("unused")
+public class BenchFlowMinioClient {
+
+    private MinioClient mc;
+    private static final String TESTS_BUCKET = "tests";
```

Comment:
I would not configure these variables here. Don't we have a better place to configure them?